Library/Homebrew/test/cmd/fetch_spec.rb

```diff
@@ -14,4 +18,27 @@
     expect(HOMEBREW_CACHE/"testball--0.1.tbz").to be_a_symlink
     expect(HOMEBREW_CACHE/"testball--0.1.tbz").to exist
   end
+
+  it "prevents double fetch (without nuking existing installation)", :integration_test, :needs_macos do
```

Comment:
Any way to do this without an integration test? They are really slow so we tend to only have one per command.

CC @reitermarkus for thoughts

Reply:
There's `Homebrew.fetch_cask` which takes a `Cask::Download`.

Reply:
Yes.